go/mysqlconn/replication/binlog_event.go

```diff
@@ -297,6 +243,17 @@ func (b *Bitmap) Bit(index int) bool {
 	return b.data[byteIndex]&bitMask > 0
 }
 
+// Set sets the given boolean value.
+func (b *Bitmap) Set(index int, value bool) {
```

Comment:
In my last review, I was wondering why this function was missing :).
I don't see unit tests for these. Are you covering them indirectly?